src/app/forms/form-common-options/form-common-options.js

```diff
@@ -0,0 +1,192 @@
+export const formCommonOptions = {
```

Comment:
👏 👏 

Nice work getting this into a shared state. Let's change it so that we export each thing individually instead of all in one big `formCommonOptions` object. For example:

```js
export const petitionerRepresentativeOptions = [...]

export const countyOptions = [...]

export const courtTypeOptions = [...]
```